_posts/2019-05-16-released-v6.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Released Chainer/CuPy v6.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v6.0.0 today!
+This is a major release that introduces several new features.
+Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tag/v6.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v6.0.0).
+
+## ChainerX
+
+The biggest update is the introduction of **ChainerX**.
+It is a fast and portable ndarray engine with autograd support written in C++ with very thin Python wrapper.
+
+We have released the beta version of ChainerX in v6.0.0b1 as we wrote in the [previous blog post](https://chainer.org/announcement/2018/12/03/chainerx.html).
+Since then, we have been working on improving it in various aspects.
+In particular, ChainerX in v6.0.0 expands the coverage of various features since v6.0.0b1.
+
+- **More op coverage**.
+  We have more and more Chainer functions that directly call ChainerX's low-overhead implementation.
+  The effort is still on going at [the tracking issue](https://github.com/chainer/chainer/issues/6423) with [the spreadsheet of op-wise implementation status](https://docs.google.com/spreadsheets/d/1B4E78tw9Awgpcdn5G7zsQ8NVFYJdOoJlIQg42QxKNfU).
+  We continue expanding the op coverage towards next v7 release.
+  Contributions are always welcomed!
+- **More Function coverage**.
+  Most users will start using ChainerX through Chainer's existing interface (just by replacing NumPy/CuPy array with ChainerX array).
+  When ChainerX does not have an implementation for an operation, Chainer automatically falls back to NumPy/CuPy-based implementation.
+  It basically works without any fix for most functions, but sometimes not.
+  We are fixing such bugs to enlarge the coverage of functions for ChainerX usage.
+  The effort is accompanied with the introduction of new simplistic test fixtures for function tests (you can find [the tracking issue](https://github.com/chainer/chainer/issues/6071)).
```

Comment:
I'm not too familiar with "simplistic" but is it suitable here? Otherwise, I'd think of.
```suggestion
  The effort is accompanied by the introduction of a test fixture class for function tests  (you can find [the tracking issue](https://github.com/chainer/chainer/issues/6071)).
```